aws/cloudformation-templates/template.yaml

```diff
@@ -174,6 +177,13 @@ Parameters:
       is set to 'Yes'.
     Default: "AWS Retail Demo Store"
   
+  PinpointSMSLongCode:
+    Type: String
+    Description: >
+      Phone number(a dedicated long code) for Amazon Pinpoint to send and receive messages at. Be sure to enable two way SMS for this long code using Pinpoint. This parameter only applies when 'Auto-Configure Pinpoint'
```

Comment:
Can we point them to the workshop/long code aws doc to give them more information?

Reply:
Sure I can add a link to aws doc